.github/ISSUE_TEMPLATE/Guidance_Issue.md

```diff
@@ -1,9 +1,10 @@
 ---
 name: Community Guidance Request ✨
```

Comment:
but the ✨ boi survived?

Reply:
Didn't need to modify this template so I think GH left the emoji alone.

Reply:
Yeah, odd. Something must've happened in the GH UX.